tier4/{{cookiecutter.project_slug}}/.github/codejson/{{cookiecutter.project_name}}/code.json

```diff
@@ -33,6 +33,8 @@
   "localisation": "{{ cookiecutter.localisation }}",
   "repoType": "{{ cookiecutter.repo_type }}",
   "userInput": "{{ cookiecutter.user_input }}",
+  "usageType": "{{ cookiecutter.usage_type }}",
```

Comment:
Please add `usageType` and `exemptionText` inside the `permissions` object as shown in the code.json schema here:
https://github.com/GSA/code-gov-data/blob/master/schemas/schema-2.0.0.json
https://github.com/GSA/code-gov/blob/master/docs/metadata_examples.md